.github/workflows/release.yml

```diff
@@ -1,9 +1,12 @@
 name: Release CI
 
 on:
+  workflow_dispatch:
```

Comment:
Just curious as to if we want to permanently add manual workflow runs as an option or if this was a temporary thing.